service.js

```diff
@@ -14,7 +14,7 @@ module.exports = (models, modelName) => ({
     let updates = await Promise.all(_.map(async association => {
       // Get Child Info
       let childRecord = record[association.alias]
-      if (!childRecord) return {}
```

Comment:
Sure, https://github.com/smartprocure/sails-nested-blueprint/issues/3
Can we merge it before the unit testing? because this is an impediment right now